src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+
+    {person.age !== undefined ? (
+      <p className="Person__age">I am {person.age}</p>
+    ) : null}
+
+    <p className="Person__partner">
+      {person.isMarried === false
+        ? 'I am not married'
+        : `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
Ternary operator inside ternary operator is a bad practice, create a variable for this value to avoid it